src/lib/tvgPictureImpl.h

```diff
@@ -39,6 +39,8 @@ struct Picture::Impl
     void *rdata = nullptr;              //engine data
     float w = 0, h = 0;
     bool resizing = false;
+    float vw = 0, vh = 0, vx = 0, vy = 0;  //viewbox parameters
```

Comment:
i think we can avoid this memory allocation.. guess duplicate could share the source's loader like flyweight.

ie) tvgPictureImpl.h

    Paint* duplicate()
    {
        ...
         dup->loader = loader;
        ...
    }